File: packages/expo-face-detector/android/src/main/java/expo/modules/facedetector/FaceDetectorPackage.java
package expo.modules.facedetector;

import android.content.Context;

import expo.modules.core.BasePackage;
import expo.modules.core.ExportedModule;
import expo.modules.core.interfaces.InternalModule;

import java.util.Collections;
import java.util.List;

public class FaceDetectorPackage extends BasePackage {
  @Override
  public List<InternalModule> createInternalModules(Context context) {
    return Collections.singletonList((InternalModule) new ExpoFaceDetectorProvider());
  }

  @Override
  public List<ExportedModule> createExportedModules(Context reactContext) {
    return Collections.singletonList((ExportedModule) new FaceDetectorModule(reactContext));
  }
}

File: android/versioned-abis/expoview-abi47_0_0/src/main/java/abi47_0_0/expo/modules/facedetector/FaceDetectorPackage.kt
package abi47_0_0.expo.modules.facedetector

import android.content.Context
import abi47_0_0.expo.modules.core.BasePackage

class FaceDetectorPackage : BasePackage() {
  override fun createInternalModules(context: Context) =
    listOf(ExpoFaceDetectorProvider())

  override fun createExportedModules(context: Context) =
    listOf(FaceDetectorModule(context))
}
